.eslintrc.cjs

```diff
@@ -0,0 +1,85 @@
+const lernaJSON = require('./lerna.json');
+
+module.exports = {
+  extends: ['airbnb', 'prettier'],
+  plugins: ['import', 'prettier'],
+  env: {
+    browser: true,
+    jest: true,
+  },
+  rules: {
+    'arrow-body-style': 'off',
+    'arrow-parens': ['error', 'as-needed'],
+    'import/no-relative-packages': 'off',
+    'import/prefer-default-export': 'off',
+    'import/extensions': 'off',
+    camelcase: 'off',
+    indent: [
+      'error',
+      2,
+      {
+        SwitchCase: 1,
+        ignoredNodes: [
+          'TaggedTemplateExpression[tag.name=/html|outdent/]',
+          'TaggedTemplateExpression[tag.name=/html|outdent/] *',
+        ],
+      },
+    ],
+    'no-confusing-arrow': 'off',
+    'class-methods-use-this': 'off',
+    'no-return-assign': 'off',
+    'no-bitwise': 'off',
+    'no-param-reassign': ['error', { props: false }],
+    'no-plusplus': ['error', { allowForLoopAfterthoughts: true }],
+    'no-underscore-dangle': 'off',
+    'no-use-before-define': ['error', { functions: false, classes: false }],
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: [
+          'scripts/**',
+          'rollup.config.js',
+          '**/story.{js,jsx}',
+          '**/demo.{js,jsx}',
+          '**/react/*.{js,jsx}',
+          '**/05-utils/**/*.{js,jsx}',
+          '**/src/**/*.react.js',
+          '**/demo/**',
+          '**/ui.test.js',
+          '**/unit.test.{js,jsx}',
+          '**/*test.accessor.js',
+          '**/*ui.test.js',
+        ],
+        packageDir: [__dirname, ...lernaJSON.packages],
```

Comment:
Maybe there need to test this on Windows

Reply:
Yes, that would be good. Added to checklist...